### PR TITLE
#93 Add "Open with Gitpod" Integration

### LINKED ISSUE #93: Configure an "Open with Gitpod" button
**Is your feature request related to a problem? Please describe.**
In the README.md file having a button, which allows the contributors to open the project via gitpod will be very helpful to new developers seeking to contribute to the project

**Describe the solution you'd like**
Update the repository to add the "open with Gitpod" button and configure it to run the project in gitpod after installing required dependencies.

**Additional context**
image

- [x] Would you like to work on this?

### PR DESCRIPTION
**Pull request by void-ness**
- [x] Updated the README file to add a button, clicking which will open the project in a Gitpod workspace. Adding such feature will be helpful for first time contributors who are facing trouble while running the project on their local machines.  

image

- [x] Updated the src path of the image in the footer. To refer to an image stored in the public directory, PUBLIC URL needs to be referred to. 

Closes #93

### CODE REVIEW

File: src/Footer.js [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
       behavior: 'smooth'
     });
   };
   window.addEventListener('scroll', toggleVisible);
 
-    return (
-      <footer className="footer">
-        <div className="top" onClick={scrollToTop}>
-              Back to top
+  return (
```
Comment (Kanika637) [inline src/Footer.js:27]: @void-ness , Could you please add a screenshot of the changes you have done in the footer?

Comment (void-ness) [reply src/Footer.js:27]: Original | Updated
 :--------------------: |:--------------------:
image | image

Earlier the image was not loading due to incorrect value in src attribute for the image.
https://github.com/Kanika637/amazon-clone/blob/6a789738ecdb8a35437753b3017e8ca8a5c017d1/src/Footer.js#L127-L129

Here's the updated code snippet
image

I have a prettier extension installed which might have increased the number of changes in the given file 😅

**Review by Jai-Solania-29 — APPROVED** [review]
LGTM 🚀

**Comment by void-ness** [comment]
Thank you! If you don't mind can you please add a hacktoberfest-accepted label over it? It currently says the PR is not accepted. @Kanika637

**Pull request merged**